proxy-node-shared/src/main/java/uk/gov/ida/notification/saml/ResponseAssertionDecrypter.java

```diff
@@ -18,10 +16,6 @@
 
     public ResponseAssertionDecrypter(Credential credential) {
         this.decrypter = new DecrypterFactory().createDecrypter(Collections.singletonList(credential));
-        if (credential.getEntityId() != null && credential.getEntityId().equals(CloudHsmCredentialConfiguration.ID)) {
-            ProxyNodeLogger.info("Using CloudHSM so set JCA provider to Cavium");
-            this.decrypter.setJCAProviderName("Cavium");
```

Comment:
Are we sure we no longer need this? The translator also uses the `ResponseAssertionDecrypter` (maybe)?

Reply:
(Nah, that's only in a test, and has a non-HSM key as well.)